Source/Core/Core/FifoPlayer/FifoPlayer.cpp

```diff
@@ -189,6 +189,18 @@ bool FifoPlayer::IsRunningWithFakeVideoInterfaceUpdates() const
   return m_File->ShouldGenerateFakeVIUpdates();
 }
 
+u32 FifoPlayer::GetMaxObjectCount() const
+{
+  u32 result = 0;
```

Comment:
Nothing wrong with what you have but here's an alternative:

```
auto comparison = [](const AnalyzedFrameInfo& a, const AnalyzedFrameInfo& b)
{
  return a.objectStarts.size() < b.objectStarts.size();
 };

return std::max_element(m_FrameInfo.begin(), m_FrameInfo.end(), comparison);
```

Reply:
I don't think that would work.  It'd need to dereference the iterator and then look at that value's `objectStarts.size()`, and I'm not 100% sure that `m_FrameInfo` can't be empty at this point so it'd also need to check that it's not `m_FrameInfo.end()`.  That'd just make it a bunch more complicated; I don't think it's worth it.